core/transport/src/grpc/file_system_master.proto

```diff
@@ -184,6 +184,7 @@ message ListStatusPOptions {
   optional LoadMetadataPType loadMetadataType = 2;
   optional FileSystemMasterCommonPOptions commonOptions = 3;
   optional bool recursive = 4;
+  optional bool includeUfsInfo = 5;
```

Comment:
Do we want to default this to `true` for backward compatibility?

Reply:
https://github.com/Alluxio/alluxio/pull/10006/commits/5551bc48c4ce46c8170c1c202af7f80e86d31abd

Reply:
We cannot set to true at proto layer.
However made a fix to interpret absence of this field properly for supporting older clients.

Reply:
if `false` is default, then we should reverse this to be `excludeUfsInfo`

Reply:
That would conform with the PR title as well.